base_site/index.html

```diff
@@ -1,39 +1,50 @@
-<!DOCTYPE html><!--  This site was created in Webflow. https://webflow.com  --><!--  Last Published: Thu Aug 01 2024 22:06:26 GMT+0000 (Coordinated Universal Time)  -->
+<!DOCTYPE html><!--  This site was created in Webflow. https://webflow.com  --><!--  Last Published: Tue Aug 27 2024 20:38:57 GMT+0000 (Coordinated Universal Time)  -->
 <html data-wf-page="668ee46537559fbfc2a09c1d" data-wf-site="668ee46537559fbfc2a09c17">
 <head>
   <meta charset="utf-8">
-  <title>Quantinuum Technical Documentation</title>
+  <title>Docs</title>
```

Comment:
```suggestion
  <title>Quantinuum Technical Documentation</title>
```

I think we need to feed this one back to Luke to keep in as a change so we don't have to revert it with every merge - "Docs" is too vague as a browser window/tab title.